bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/Higgs/thq_4f_ckm_LO/thq_4f_ckm_LO_proc_card.dat

```diff
@@ -0,0 +1,15 @@
+set group_subprocesses Auto
+set ignore_six_quark_processes False
+set loop_optimized_output True
+set complex_mass_scheme False
+define p = g u c d s u~ c~ d~ s~
```

Comment:
please remove proton and jet definitions and follow these recommendations https://twiki.cern.ch/twiki/bin/viewauth/CMS/Moriond18MC#Checks_to_the_datacards_BEFORE_T